internal/main.go

```diff
@@ -177,25 +89,25 @@ func RunTests(opts RunTestsOpts) error {
 		return err
 	}
 
-	// channel to receive each 'go test' stdout line
-	goTestOutput := make(chan TestEvent)
+	var failedTests []string
 
 	var coverProfile string
-	if opts.FlagCoverReport || opts.FlagCoverProfile != "" {
-		coverProfile = zvfb(opts.FlagCoverProfile, "./coverage.out")
-	} else if opts.FlagFullCoverage {
-		newF, err := os.CreateTemp("", "coverage-*.out")
-		if err != nil {
-			return err
+	if opts.FlagCoverReport || opts.FlagFullCoverage {
```

Comment:
These are the only cases we need `coverProfile`.
Only set in these cases so cached tests can be used.